package/setup.py

```diff
@@ -558,6 +558,7 @@ def dynamic_author_list():
           'scipy>=1.0.0',
           'matplotlib>=1.5.1',
           'mock',
+          'tidynamics>=1.0.0',
```

Comment:
Should minimum dependencies be specified with `conda` as well?

https://github.com/MDAnalysis/mdanalysis/blob/4ca2bf7a96f0f6c7c4859240db0a4333f36b3830/.appveyor.yml#L13

https://github.com/MDAnalysis/mdanalysis/blob/4ca2bf7a96f0f6c7c4859240db0a4333f36b3830/.travis.yml#L30

Reply:
Should tidynamics be packaged under the `analysis` dependencies, or under `install_requires` or under `requires` or some combination thereof?

I only ask due to `tqdm` being packaged under `install_requires` and `requires` despite not being on the core dependency list (at least in the user guide).

If not installed/required by default the `import tidynamics` should be `try/except` guarded and `_conclude_fft()` method should raise `ImportError`